assets/custom.js

```diff
@@ -10,8 +10,16 @@ function activate_sidebar() {
 function activate_dropdown() {
```

Comment:
simplify your code to this one

```$(document).ready(function(){
	$('.ui.sidebar').sidebar('toggle');
	$('.dropdown').dropdown();
});```

add class `dropdown` to `#project_filter`, `#feed1` and `#feed2`